packages/@aws-cdk/aws-apigatewayv2/lib/websocket/api.ts

```diff
@@ -18,6 +18,24 @@ export interface IWebSocketApi extends IApi {
   _addIntegration(scope: Construct, config: WebSocketRouteIntegrationConfig): WebSocketIntegration
 }
 
+/**
+ * Represents the currently available API Key Selection Expressions
+ */
+export enum ApiKeySelectionExpression {
```

Comment:
Use an enum-like class here instead of an enum - https://github.com/aws/aws-cdk/blob/master/docs/DESIGN_GUIDELINES.md#enums